### PR TITLE
Fixing phishing attachments bug

### PR DESCRIPTION
**Pull request by tomer-pan**
## Related Issues
fixes: https://jira-hq.paloaltonetworks.local/browse/XSUP-14256

## Description
A fix of a breaking change made in an earlier version of the phishing pack.

## Minimum version of Cortex XSOAR
- [x] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

### CODE REVIEW

**Comment by tomer-pan** [comment]
@idovandijk Created this PR since the other one got messed up (look at the commits dates - it look like they are old but they are not).
Please approve this (I will merge this instead).

**Review by ssokolovich — APPROVED** [review]

**Review by idovandijk — COMMENTED** [review]
Please add the phishing tests to the YML file of the Process Email v2 pb, since the current test is testing the Process Email v1

File: Packs/Phishing/Playbooks/Process_Email_-_Generic_v2.yml [hunk 1]
```diff
@@ -2463,5 +2480,6 @@ outputs:
   description: The file object.
   type: string
 tests:
-- Process Email - Generic - Test - Incident Starter
-fromversion: 6.0.0
\ No newline at end of file
+- Phishing v3 - Get Original Email + Search & Delete - Test
```
Comment (idovandijk) [inline Packs/Phishing/Playbooks/Process_Email_-_Generic_v2.yml:2483]: Great, we used to have 0% coverage of this playbook

**Review by idovandijk — APPROVED** [review]
Great

**Pull request merged**